internal/cmd/perfcomp/main.go

```diff
@@ -305,27 +318,31 @@ func getEnergyStatsForAllBenchMarks(ctx context.Context, patchRawData []RawData,
 
 func generatePRComment(energyStats []*EnergyStats, version string) string {
 	var comment strings.Builder
-	comment.WriteString("# 👋GoDriver Performance\n")
-	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n", version)
+	fmt.Fprintf(&comment, "The following benchmark tests for version %s had statistically significant changes (i.e., |z-score| > 1.96):\n\n", version)
 
 	w := tabwriter.NewWriter(&comment, 0, 0, 1, ' ', 0)
 	fmt.Fprintln(w, "| Benchmark\t| Measurement\t| H-Score\t| Z-Score\t| % Change\t| Stable Reg\t| Patch Value\t|")
```

Comment:
Optional: I suspect many people are not familiar with H and Z scores in this context. Consider linking those column headings to explanations of what the H and Z score are.